packages/cli/src/commands/apps/delete.ts

```diff
@@ -11,16 +11,17 @@ export default class AppDeleteCommand extends APICommand {
 		description: 'App profile UUID or number in the list',
 	}]
 
-	primaryKeyName = 'appId'
-	sortKeyName = 'displayName'
-
 	async run(): Promise<void> {
 		const { args, argv, flags } = this.parse(AppDeleteCommand)
 		await super.setup(args, argv, flags)
 
-		await selectAndActOn(this, args.id,
-			async () => await this.client.apps.list(),
-			async id => { await this.client.apps.delete(id) },
-			'app {{id}} deleted')
```

Comment:
I think a majority of these output messages in commands were already lowercase. We may want to double check that we aren't introducing inconsistency by only changing a subset of them here.